FastImageCache/FastImageCache/FastImageCache/FICUtilities.m

```diff
@@ -44,10 +44,8 @@ CFUUIDBytes FICUUIDBytesWithString(NSString *string) {
     CFUUIDBytes UUIDBytes;
     CFUUIDRef UUIDRef = CFUUIDCreateFromString(kCFAllocatorDefault, (CFStringRef)string);
     
-    if (UUIDRef != NULL) {
-        UUIDBytes = CFUUIDGetUUIDBytes(UUIDRef);
-        CFRelease(UUIDRef);
-    }
+    UUIDBytes = CFUUIDGetUUIDBytes(UUIDRef);
```

Comment:
Currently wants that UUIDBytes can be used uninitialized in the else case. Inspected CFUUIDCreateFromString, which does not return an optional so no need to wrap in if case.